airflow/www/views.py

```diff
@@ -2307,7 +2307,7 @@ def version(self):
         # Get the Git repo and git hash
         git_version = None
         try:
-            with open("airflow/git_version") as f:
+            with open(os.path.join(*[settings.AIRFLOW_HOME, 'airflow', 'git_version'])) as f:
```

Comment:
Curious about this. Do you guys run with AIRFLOW_HOME containing the entire AIRFLOW code base? We `pip install` airflow, and AIRFLOW_HOME points to a separate directory that just has airflow.cfg, dags dir, etc. I'm not sure that this would work for us, since `git_version` would be under the Python package dir, not AIRFLOW_HOME.